packages/crew_profile.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Crew_profile < Package
+  description 'A profile for Chromebrew, to be used instead of ~/.*rc'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version "#{CREW_VERSION}"
```

Comment:
We're gonna need a static version.  This will change when the crew version changes even if there are no changes in this package.

Reply:
OK.